### PR TITLE
feat(offline): retry server navigation when back online

### LINKED ISSUE #9207: Fusion: Weird UI behavior when offline
As a user, when I am offline, if I see the offline page, I don't want to see the Vaadin reconnection dialog and `reconnect attempt xxx for heartbeat`, see the attached screenshot.

### PR DESCRIPTION
**Pull request by platosha**
Fixes #9207

### CODE REVIEW

File: flow-tests/test-ccdm-flow-navigation/sw.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+// @ts-ignore: __WB_MANIFEST is injected by the InjectManifest plugin
+let manifestEntries: Array<PrecacheEntry> = self.__WB_MANIFEST;
+// @ts-ignore: additionalManifestEntries is defined in sw-runtime-resources-precache.js
+const additionalManifestEntries: Array<PrecacheEntry> = self.additionalManifestEntries;
+if (additionalManifestEntries && additionalManifestEntries.length) {
+  manifestEntries = [...manifestEntries, ...additionalManifestEntries];
+}
+
+precacheAndRoute(manifestEntries);
```
Comment (joheriks) [inline flow-tests/test-ccdm-flow-navigation/sw.ts:45]: Was this file intended to be added?

Comment (platosha) [reply flow-tests/test-ccdm-flow-navigation/sw.ts]: oops, accidental. Removed.

**Review by joheriks — APPROVED** [review]

**Review by haijian-vaadin — APPROVED** [review]
